features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -748,6 +748,18 @@
             {% endif %}
         </OpenIDConnect>
 
+         <!--
+            FAPI Support.
+            Supported versions: IS 5.11.0 onwards
+        -->
+        {% if fapi is defined %}
+        <FAPI>
```

Comment:
Can we rethink the configuration names?
Also wouldn't it be better to have this configuration under OpenIDConnect section?

Reply:
Added this after a discussion with @VivekVinushanth .Do you have a suggestion? This is necessary in FAPI:CIBA conformance suite and FAPI security conformance suites to add an extra validation.

Reply:
How about something like <FapiSupported>true</FapiSupported> or <EnableFapi>true</EnableFapi>

Reply:
Changed to following after discussion
[fapi]
enable_ciba_profile=true
enable_security_profile=true